articles/Angle/agEUR_post_Euler.md

```diff
@@ -0,0 +1,217 @@
+# Consequences of the Euler Exploit for Angle's agEUR and Euro-Denominated Curve Pools
+
+## Useful links
+
+* [Angle Website](https://www.angle.money/#/) | [General Docs](https://docs.angle.money/overview/readme) | [Developer Docs](https://developers.angle.money/)
+* [Angle Governance Forum](https://gov.angle.money/) | [Snapshot page](https://snapshot.org/#/anglegovernance.eth)
+* [AgEUR - LlamaRisk Assessment (Jan 2022)](https://cryptorisks.substack.com/p/ageur-angle-protocol)
+* [AIP-43: Improve Angle Yield Strategies for USDC and DAI](https://snapshot.org/#/anglegovernance.eth/proposal/0xb1b4d98c080ec587b2563a6aaa6f854e0a42ce6881f61bced62cf9fa8ae42898)
+* [Euler exploit analysys](https://medium.com/@numencyberlabs/a-detailed-analysis-of-Euler-finances-196-million-flash-loan-attack-81cdef370024)
+* [AgEUR/EUROC Factory Pool](https://curve.fi/#/ethereum/pools/factory-v2-164/deposit) | [3EURpool Factory Pool](https://curve.fi/#/ethereum/pools/factory-v2-66/deposit)
+* [Angle Protocol - Q&A Regarding Euler Exploit](https://anglemoney.notion.site/Angle-Protocol-Q-A-Regarding-Euler-Exploit-03af18cbe5e84430b3341b145554492e)
+* [Angle Protocol - Ideas for re-launching the protocol](https://gov.angle.money/t/ideas-for-re-launching-the-protocol/674)
+* [Euler Hack Reflection: Learning, Adapting, and Strengthening Angle Protocol](https://www.angle.money/#/blog/announcements/euler-hack-reflection-learning-adapting-and-strengthening-angle-protocol)
+
+## Introduction
+
+In a [previous article](https://cryptorisks.substack.com/p/ageur-angle-protocol), we explored the Angle Protocol, an innovative stablecoin framework designed for price stability, over-collateralization, and optimal capital efficiency. The protocol's stablecoin, agEUR, aims to be pegged to the Euro and has demonstrated impressive price stability compared to other Euro-based stablecoins. This report delves into the Euler Finance exploit that occurred on March 13th, 2023, examining its impact on the peg and liquidity on Curve. It also investigates the strategies employed for Angle Core Module's reserves, the crisis management measures Angle took, actions taken following Euler's recovery, and the lessons learned from the incident. Finally, the report provides an updated risk assessment for agEUR and its related Curve pools.
+
+### Key Findings:
+
+- In Q1 2023, Angle modified its reserve investment approach, enabling permissionless proposals for allocation changes and acceptance based solely on APR. However, Angle did not consider additional risk mitigation by diversifying yield sources. During the exploit, Angle had 74% of its USDC reserves invested in Euler.
```

Comment:
"enabling permissionless proposals for allocation changes and acceptance based solely on APR" statement may be rephrased, clarified